src/main/java/de/tum/in/www1/artemis/domain/enumeration/FeedbackType.java

```diff
@@ -2,9 +2,28 @@
 
 /**
  * The FeedbackType enumeration.
+ * <p>
  * The order in which they are declared is important, as we use the enums default implementation of
  * <a href="https://docs.oracle.com/javase/10/docs/api/java/lang/Enum.html#compareTo(E)">compareTo()</a>
+ * <p>
```

Comment:
Wouldn't it be better if we compared the enums by their name instead. Otherwise you cannot remove enums in the future or replace them.

Reply:
Sure, but `compareTo` is final in the `Enum` class, so we can’t override it here. Therefore, I think we would have to carefully check the code where this assumption of sorting is currently made and adapt those.

This is out of scope for this PR.